docs/elastic_inference.md

```diff
@@ -0,0 +1,182 @@
+
+11. Review the configuration of your instance and choose Launch.
+
+12. You are prompted to choose an existing key pair for your instance or to create a new key pair. 
+
+```
+WARNING
+
+    Don’t select the Proceed without a key pair option. If you launch your instance without a key pair, then you can’t connect to it.
+
+    After making your key pair selection, choose Launch Instances.
+```
+
+It can take a few minutes for the instance to be ready so that you can connect to it. Check that your instance has passed its status checks. You can view this information in the Status Checks column.
+
+
+## Custom Service file with EIA 
+
+A custom service, for MXNet that supports EIA, consists of two contexts. The input data would be loaded from cpu context and the symbol is bound to eia context, i.e:
+```text
+1. mxnet.cpu() - used for loading up input data
+2. mxnet.eia() - Used for binding network symbols and params to an attached EIA instance.
+```
+
+        
+We modify the [base model service template](https://github.com/awslabs/mxnet-model-server/blob/master/examples/model_service_template/mxnet_model_service.py) to support EIA.
+
+
+```python
+    def initialize(self, context):
+        # NOT COMPLETE CODE, refer template above for it.
+        .....
+        .....
+        .....           
+        # Load MXNet module
+        # Symbol Context set to eia
+        self.mxnet_ctx = mx.eia()
+        self.data_ctx = mx.cpu()
+        sym, arg_params, aux_params = mx.model.load_checkpoint(checkpoint_prefix, self.epoch)
+
+        # noinspection PyTypeChecker
+        self.mx_model = mx.mod.Module(symbol=sym, context=self.mxnet_ctx,
+                                      data_names=data_names, label_names=None)
+        self.mx_model.bind(for_training=False, data_shapes=data_shapes)
+        self.mx_model.set_params(arg_params, aux_params, allow_missing=True, allow_extra=True)
+    
+    def inference(self, model_input):
+        # NOT COMPLETE CODE, refer template above for it.
+        .....
+        .....
+        .....     
+        model_input = [item.as_in_context(self.data_ctx) for item in model_input]
+
+```
+
+The above code shows initialization of two contexts, one for data (on CPU) and other for symbols (on EIA).
+
+Once we have the code ready. We can build a model archive consumable by MMS, using the model-archiver. The [model-archiver](https://github.com/awslabs/mxnet-model-server/blob/master/model-archiver/README.md) tool enables the create to an archive understood by MMS.
+
+```python
+model-archiver --model-name <model-name> --handler model_service:handle --export-path <output-dir> --model-path <model_dir> --runtime python
+```
+
+This will create file ```<model-name>.mar``` in the directory ```<output-dir>```.
+
+## Running elastic inference on a resnet-152
+A prebuilt Amazon Elastic Inference supporting resnet-152  model archive can be downloaded using the following command:
+
+```bash
```

Comment:
I think we should merge all these steps into a single start cli:
mxnet-mode-server --start --models reset-152-eai=https://s3.amazonaws.com/model-server/model_archive_1.0/resnet-152-eia.mar